pkg/test/framework/suite.go

```diff
@@ -302,6 +307,39 @@ func (s *suiteImpl) RequireMaxVersion(minorVersion uint) Suite {
 	return s
 }
 
+func (s *suiteImpl) RequireDualStackCluster() Suite {
+	fn := func(ctx resource.Context) error {
+		for _, c := range ctx.Clusters().Kube() {
```

Comment:
super nit: can do this in the Environment setup as we create kube clusters _or_ cache this value on the `Cluster` struct somehow, just to avoid adding warmup time to each suite, even though it's probably very minimal.